pkgs/development/python-modules/solo-python/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildPythonPackage, fetchPypi
+, click, ecdsa, fido2, intelhex, pyserial, pyusb, requests}:
+
+ buildPythonPackage rec {
+  pname = "solo-python";
+  version = "0.0.15";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12s5lv2n02an8nscspv8s1gknh593l7mqljqvxpnbkzfjf6icfvc";
+  };
+
+  propagatedBuildInputs = [
+    click
+    ecdsa
+    fido2 
+    intelhex
+    pyserial
+    pyusb
+    requests
+  ];
+
+  # test failes with:
+  #
+  # running build_ext
+  # Usage: nix_run_setup [OPTIONS] COMMAND [ARGS]...
+  # Try "nix_run_setup --help" for help.
+  #
+  # Error: No such command "test".
+  doCheck = false;
```

Comment:
They don't package their tests are part of the sdist. I would pull from their github, tests are important to make sure the package doesn't get broken when upstream dependencies get updated

Reply:
I changed it to pull from their github, though the build fails now with:
```
  configuring
  no configure script, doing nothing
  building
  Executing setuptoolsBuildPhase
  Traceback (most recent call last):
    File "nix_run_setup", line 8, in <module>
      exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
    File "/nix/store/k0igqpz995nmxwf6piyd5jjpcc7gcnyc-python3-3.7.4/lib/python3.7/tokenize.py", line 447, in open
      buffer = _builtin_open(filename, 'rb')
  FileNotFoundError: [Errno 2] No such file or directory: 'setup.py'
```
How do I fix this?

Reply:
this is because the flit package is broken... ill send up a PR to fix it

Reply:
flit fix got merged, this should build fine now. Thanks for putting up with the trouble :)

Reply:
Thanks for the fast help!